client/app/main/student/student.controller.js

```diff
@@ -14,6 +14,16 @@ function StudentCtrl($scope, $state, $stateParams, Student, toastr, Modal) {
     $scope.student = result.student;
     $scope.interventions = result.interventions;
     $scope.outreaches = result.outreaches;
+    Student.list({id: $stateParams.id}, function(data) {
+      var record = data.currentRecord.entries;
+      var enrolled = 0;
+      var present = 0;
+      _.forEach(record, function(entry) {
```

Comment:
There should always be a single entry in record. You could validate that, but i don't think it is necessary. I would just get the record using:
```
var entry = _.first(data.currentRecord.entries); // or data.currentRecord.entries[0]
$scope.percentage = (entry.enrolled / entry.present).toFixed(2);
```